lib/client/credential/idemix/signerconfig.go

```diff
@@ -20,8 +20,6 @@ type SignerConfig struct {
 	EnrollmentID string `protobuf:"bytes,5,opt,name=enrollment_id,json=enrollmentId" json:"enrollment_id,omitempty"`
 	// CRI contains a serialized Credential Revocation Information
 	CredentialRevocationInformation []byte `protobuf:"bytes,6,opt,name=credential_revocation_information,json=credentialRevocationInformation,proto3" json:"credential_revocation_information,omitempty"`
-	// CurveID specifies the name of the Idemix curve to use, defaults to 'amcl.Fp256bn'
-	CurveID string `protobuf:"bytes,7,opt,name=curve_id,json=curveID" json:"curveID,omitempty"`
```

Comment:
CurveID was recently added by Yacov, don't we need it?

Reply:
Yes, but we don't need this field. It is never used and does not offer any advantage at the current stage.
The SignerConfig struct here is just a copy of the same struct in the Idemix lib.